aio_pika/connection.py

```diff
@@ -52,12 +52,12 @@ def __init__(self, url, loop=None, **kwargs):
         self.kwargs = self._parse_kwargs(kwargs or self.url.query)
 
         self.connection = None     # type: aiormq.Connection
-        self.close_callbacks = set()
+        self._close_callbacks = set()
```

Comment:
why you want to hide this? User may use it directly It's legal.

Reply:
It's the only place where callbacks set is public, - all other callback sets are private. I've made this to uniform callbacks storages.

Reply:
🤔

Reply:
@mosquito what do you think?

Reply:
Uniform callback storage sounds good. Let's do it.